usagov/vote-gov-drupal
# VOTE-1530: upgrade browser-sync to 2.29.3

## Jira ticket (required)
[VOTE-1530](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1530&assignee=609161faf6c0960069bf61b3)

## Description (optional)
upgrade browser-sync to 2.29.3

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. lando retune

### Testing steps
1. run command 'npm run dev' inside theme directory
2. Confirm no dependency errors
